.github/workflows/http_crawler_docker_release.yml

```diff
@@ -0,0 +1,21 @@
+name: HTTP Crawler Docker Image CI
+
+on:
+  push:
+      branches: [ "main" ]
```

Comment:
I don't think we want to push on every push to main. I think we should either only run on version tags (v.*), or restrict the action to run only if the crawler, poetry.lock, pyproject.toml, or README.md files have changed.

So https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-branches-and-tags
or https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths

Same comment for other build.

I'm slightly leaning toward running on a version because that what I expect we'll do for sycamore.